data/hai/midas.txt

```diff
@@ -0,0 +1,993 @@
+				outfit "Gold Bars" -5
+	# accept is used when you spot the crew member but do not know the link to the bars. If they are revealed, they will not try again.
+	on accept
+		fail "Midas: Betrayal For The Bars"
+
+# Sell the bars if you agreed to the deal, 1 million a piece and 5 by 5 followed by one by one so we can get all of them.
+mission "Midas: Bar Selling"
+	minor
+	landing
+	invisible
+	repeat
+	substitutions
+		"<amount>" "one"
+		"<amount>" "five"
+		"<amount>" "ten"
+	to offer
+		has "midas: sold bars"
+		has "outfit: Gold Bars"
+	on offer
+		conversation
+			action
+				payment 1000000
+				outfit "Gold Bars" -1
+			branch selling
+				"outfit: Gold Bars" < 4
+			action
+				payment 4000000
+				outfit "Gold Bars" -4
+			label selling
+			`An operative of the organization you're selling to offers you the usual one million per stack of gold bars, for <amount> of your gold bars. It doesn't seem like you have much choice in the matter, so you hand them over.`
+				decline
+
+
+
+# SPOILERS: Midas want that governor down because she can easily help the Hai-Unfettered Hai unite, given she understands the Unfettered well.
+# They will only take her hostage here, she may come back later.
+mission "<uhai> 6 B: To The Wolves"
+	landing
+	name "Feed them to the Wolves"
+	description "Instead of escorting the governor as planned, take their fleet to <waypoints>, for an ambush. You may disable the governor's ships too, but risk angering the Hai by doing so."
+	source "Greenbloom"
+	destination "Warfeed"
+	to offer
+		has "<uhai> 6 A: Loyal Escort: active"
+		has "midas: tracker relay"
+		not "event: mountaintop ascendant"
+	on offer
+		dialog
+			`You are alone in your quarters, waiting for the security detachment of the Hai admiral to arrive, when you hear a mysterious voice.`
+			`	"She of the Void requests that you bring the Hai ships to <waypoints> without delay, in order to preserve Unfettered integrity. Do so and you will be richly rewarded.`
+			`	"Worry not, they will not suspect you for taking a small detour. We will bring the supplies to the Unfettered ourselves, we are only acting in their best interest, after all.`
+			`	"Whatever you decide, you do not want to speak of us to them," they add in a very serious tone, before the transmission stops. You notice a contract on your desk computer, ready to be accepted. This clearly conflicts with the official deal that was concluded between the Unfettered and the Hai, so you will have to choose between the two.`
+			``
+			`	Do you accept this deal from 'She of the Void', or do you want to escort the governor as planned?`
+	waypoint "Heia Due"
+	on waypoint
+		"reputation: Midas" >?= 0
+		conversation
+			`One of the unindentified ships hail you. "You did well, Captain. Here is the first part of your reward: <payment>. You may help us by taking out these Hai ships, but it may be wise not to so that the rest of the Hai government will not suspect you for this unfortunate pirate attack.`
```

Comment:
```suggestion
			`One of the unidentified ships hail you. "You did well, Captain. Here is the first part of your reward: <payment>. You may help us by taking out these Hai ships, but it may be wise not to so that the rest of the Hai government will not suspect you for this unfortunate pirate attack.`
```